frontend/src/locales/scripts/en.json5

```
@@ -498,6 +498,11 @@
     },
     imageInfo: "Image information",
     audioInfo: "Audio information",
+    tags: {
+      title: "Tags",
+      showAll: "Show all tags",
+      seeLess: "See less",
```

```suggestion
      showAll: "Show all tags",
      showLess: "See less",
```

I wonder if these keys using the same word, even though the translations do not, would make the relationship of these strings clearer to translators.

---

I think you are right, "show" everywhere, in the strings and keys, is probably the best solution.